sdcm/cluster.py

```diff
@@ -412,6 +412,9 @@ def init(self) -> None:
         self._init_port_mapping()
 
         self.set_keep_alive()
+        # Remove bash core which generated by scylla_setup for internal test purpose.
+        # Workaround for https://github.com/scylladb/scylla/issues/6159
+        self.remoter.run("sudo rm /var/lib/systemd/coredump/core.bash.*", ignore_status=True)
```

Comment:
I think it would be better to add this before `self.start_task_threads()`, to make sure it's delete before the backgroud task has a change to catch it.

Reply:
done